dtos/auth_models.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+from pydantic import BaseModel, validator, EmailStr
+from fastapi import HTTPException
+from datetime import datetime
+
+class UserLogin(BaseModel):
+    email: EmailStr
+    password: str
+    created_at: datetime
+    updated_at: datetime
```

Comment:
You can remove created_at and updated_at from UserLogin as UserLogin is for login purpose.